### PR TITLE
Color changes and rearrange styles.css

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,24 +49,6 @@ <h2 id="contact">Contact</h2>
     <ul class="inline-flex">
       <li><span class="bold">Phone: </span><br>+34 642 104 722</li>
```
Comment (Quaquaro) [inline index.html:50]: Here for there is also href="tel:...

File: index.html [hunk 2]
```diff
@@ -29,8 +29,8 @@ <h3 id="job">student web developer</h3>
       </ul>
     </nav>
     <figure>
-      <img src="images/daniel-eicher.png" alt="" width="500" height="500">
-      <figcaption>welcome to my personal web page</figcaption>
+      <img src="images/daniel-eicher.png" alt="" width="700" height="500">
```
Comment (Quaquaro) [inline index.html:32]: Why do you leave the alt tag empty?

File: styles.css [hunk 3]
```diff
@@ -51,14 +43,22 @@ section {
...
+#interests {
+  padding-bottom: 0.5rem;
 }
 
 #biography {
   padding-bottom: 2rem;
 }
 
+#tech-stack {
+  background: linear-gradient(to bottom, rgba(28, 87, 126, 0.8), rgba(207, 236, 255, 0.8));
```
Comment (Quaquaro) [inline styles.css:59]: Nice effect!

File: styles.css [hunk 4]
```diff
@@ -188,5 +209,5 @@ footer > a {
 }
 
 footer > a:hover {
-  color: (245, 252, 252);
+  color: white;
```
Comment (Quaquaro) [inline styles.css:212]: Maybe you could add for the icons that they take an opacity value of 0.5 when hovering.

**Review by Quaquaro — COMMENTED** [review]
That already looks really professional.Good choice of colours. The Montserrat font is one of my favourites.
Have a nice weekend and we'll see you on Monday.
P.S.: You have definitely earned the weekend ;)

File: index.html (hunk 2 above)
Comment (codingoak) [reply index.html:32]: oh, i forgot to fill-in information when i change the random picture - thx a lot

File: styles.css (hunk 3 above)
Comment (codingoak) [reply styles.css:59]: yes - without this effect the vs-code logo was not good appear

File: styles.css (hunk 4 above)
Comment (codingoak) [reply styles.css:212]: very nice idea, i will try to make it

File: index.html (hunk 1 above)
Comment (codingoak) [reply index.html:50]: thx for the tipp!

**Pull request merged**